machinery.spec.erb

```diff
@@ -113,22 +113,26 @@ bundle install --standalone --local
 
 rm -rf vendor .bundle Gemfile Gemfile.lock
 
-# remove Nokogiri build logs to prevent the following RPMLint error:
+# Remove references to the build directory to prevent the following RPMLint error:
 # E: file-contains-buildroot (Badness: 10000)
 rm -rf bundle/ruby/%{rb_ver}/extensions/*/%{rb_ver}/nokogiri-*/mkmf.log
 rm -rf bundle/ruby/%{rb_ver}/extensions/*/%{rb_ver}/nokogiri-*/gem_make.out
+# the nokogiri gem is mentioned twice in the setup.rb in Tumbleweed (once with
```

Comment:
Ok, to make the patch perfect start this sentence with an upper-case letter as well. Then it looks good to me.